apps/freeablo/faworld/actor.h

```diff
@@ -209,7 +210,7 @@ namespace FAWorld
 
             bool mIsDead = false;
             bool mCanTalk = false;
-            bool mIsEnemy;
+            Faction* mFaction;
```

Comment:
I had it as a regular variable before but changed it to a pointer as I had some troubles with initialization.  I agree it would be better, I will try to change the code back. (I don't remember what the problem was :D I've been hacking on this since 10 AM till now hehe)